code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1270,3 +1270,20 @@ datum/reagent/romerol
 	M.resize = 1/current_size
 	M.update_transform()
 	..()
+	
+/datum/reagent/hormone
+	name = "Exotic Hormone"
+	id = "hormone"
+	description = "Swaps your Gender"
+	reagent_state = LIQUID
+	color = "#78FFF0"
+	metabolization_rate = 5
+
+/datum/reagent/hormone/on_mob_delete(mob/m)
```

Comment:
All reagents should be typecasted to `mob/living/carbon/M` because that's the only thing that `on_mob_life`, and `on_mob_delete` can ever proc on. Also don't make it lowercase, that's different from every single other reagent in the entire codebase.